packages/ui/src/TapToEdit.tsx

```diff
@@ -122,10 +121,10 @@ export const TapToEdit = ({
           <Field
             helperText={description}
             label={title}
-            placeholder={placeholder}
+            type={(fieldProps?.type ?? "text") as NonNullable<FieldProps["type"]>}
```

Comment:
Did not know this was a thing!

Reply:
It's very rare I've run into it, but super helpful here! The real fix is probably forcing type to always be provided, but I couldn't convince the interface to do that.